models/espresso/5.4.0.yml

```diff
@@ -1 +1 @@
-include: 5.2.1.yml
+!include "models/espresso/5.2.1.yml"
```

Comment:
Does this have to be an absolute path?

Reply:
I'm afraid so, yes. The paths are relative to the current working directory of the process. And the `yaml.Type` class does not have the file path available ([source](https://github.com/nodeca/js-yaml/blob/0d3ca7a27b03a6c974790a30a89e456007d62976/lib/type.js#L5)), which I would need to use paths relative to the file.
There is a [yaml-include package](https://github.com/claylo/yaml-include), where one needs to define a base file to handle relative paths. And it seems to me it can only handle one base file, i.e. one reference, so I'm not sure if or how it would work with multiple references.
The code.js `!include` is also capable of resolving objects within a file, like so `!include someFile.yml#/some/property`.